docs/config.md

```diff
@@ -1,6 +1,6 @@
 ## Config format
 
-This document provides examples for the .yaml format for specifying rules to sgrep. It does not cover the patterns themselves; for information on the query sytnax see [patterns.md](patterns.md)
+This document provides examples for the .yaml format for specifying rules to sgrep. It does not cover the patterns themselves; for [the pattern syntax see patterns.md](patterns.md)
 
 If you want run these rules or ship them to your team, rather than write them, check out the easy-to-use [bento.dev](https://bento.dev)
```

Comment:
```suggestion
If instead of writing rules, you want run them, ship them to your team, or enforce them in CI/CD, check out the easy-to-use [bento.dev](https://bento.dev)
```

check my commas here, not quite sure if I got them right